scripts/check_frontend_coverage_test.py

```diff
@@ -204,14 +201,11 @@ def test_check_coverage_changes_insertion(self):
             with fully_covered_tests_swap:
                 with self.assertRaises(SystemExit) as error:          
                     check_frontend_coverage.check_coverage_changes()
-                self.assertEqual(error.exception.code, 1)
-                self.assertEqual(
-                    self.check_function_calls_counter['mock_print'], 4)
-                self.assertNotEqual(self.print_messages.index(
-                    ('\033[1mfile2.ts\033[0m file is fully covered but it\'s'
-                    ' supposed to be in the whitelist. Please add the file name'
-                    ' in the whitelist in the file'
-                    ' scripts/check_frontend_test_coverage.py.\n')), -1)
+                self.assertEqual(error.exception.message,
```

Comment:
When something in parens spans multiple lines, break after the opening paren. Ditto elsewhere -- please fix throughout.

Also, same thing, use assertRaisesRegexp instead of assertRaises. If your error message is long, store it in a variable: expected_error_message or something like that.

Reply:
Done.